src/datapath.jl

```diff
@@ -80,6 +89,54 @@ function TSdataset(filepath::String)
     end
 end
 
+"""
+   `load_ts_file(fpath)`
+`load_ts_file` takes the path to `ts` files and returns the table (IndexedTable)
+and Array, where `fpath` is path to `ts` file located on your computer.
+eg
+```julia
+   table_a, arr = load_ts_file("/Users/your_user_name/../GunPoint_TRAIN.ts")
+```
+"""
+function load_ts_file(fpath)
```

Comment:
add boolean kwarg argument `return_array=False` to either return array or table

Reply:
I have added kwarg argument.